.github/workflows/eslint.yml

```diff
@@ -0,0 +1,27 @@
+name: Javascript ESLint check
+
+on: [push]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12', '14' ]
```

Comment:
we do we run linting for both versions?

Reply:
it was designed and built for `12` (https://github.com/CSCfi/swift-browser-ui/blob/devel/Dockerfile#L1) but I want to see if we can easily move to newer versions such as `14` without breaking anything.

At some point will move directly to `14`, till then will be testing both.